.ci/gpg/create-keyring.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+# Modified from https://blogs.itemis.com/en/secure-your-travis-ci-releases-part-2-signature-with-openpgp
+
+function err_exit() {
```

Comment:
hi @estroz, 

You are testing it pushing a tag against your fork. Then, when the tag be pushed it should trigger the CI Travis and do the release steps.  Am I right?  Could you link your test?

Reply:
https://github.com/operator-framework/operator-sdk/pull/4034#issuecomment-721381763